release/release_gold.sh

```diff
@@ -110,23 +110,58 @@ function tag_release {
 		repository=liferay-portal
 	fi
 
-	if (! curl \
-			"https://api.github.com/repos/liferay/${repository}/git/tags" \
-			--data-raw '
-				{
-					"message": "",
-					"object": "'$(lc_get_property release-data/release.properties git.hash.liferay-portal-ee)'",
-					"tag": "'${LIFERAY_RELEASE_VERSION}'",
-					"type": "commit"
-				}' \
-			--fail \
-			--header "Accept: application/vnd.github+json" \
-			--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
-			--header "X-GitHub-Api-Version: 2022-11-28" \
-			--max-time 10 \
-			--request POST \
-			--retry 3 \
-			--silent)
+	local username=liferay
+
+	if [ -n "${LIFERAY_RELEASE_USER_REPOSITORY}" ]
+	then
+		username=${LIFERAY_RELEASE_USER_REPOSITORY}
+	fi
+
+	local response_tag_creation=$(curl -i \
+		"https://api.github.com/repos/${username}/${repository}/git/tags" \
+		--data-raw '
+			{
+				"message": "",
+				"object": "'$(lc_get_property release-data/release.properties git.hash.liferay-portal-ee)'",
+				"tag": "'${LIFERAY_RELEASE_VERSION}'",
+				"type": "commit"
+			}' \
+		--fail \
+		--header "Accept: application/vnd.github+json" \
+		--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
+		--header "X-GitHub-Api-Version: 2022-11-28" \
+		--max-time 10 \
+		--request POST \
+		--retry 3 \
+		--silent \
+		| awk '/^HTTP/{print $2}')
```

Comment:
@lucasmiranda0, can you please give me more details about this code? Why do you need this?

Reply:
The previous verification was not working well on my tests, so I started to verify based on the HTTP response. But I can recheck the previous behavior.